lib/client/kadira.js

```diff
@@ -6,17 +6,49 @@ Kadira.disableErrorTracking = function () {
   Kadira.options.enableErrorTracking = false;
 };
 
-Kadira.trackError = function (type, message, { stacks = '' } = {}) {
-  if(Kadira.options.enableErrorTracking && type && message) {
+Kadira.trackError = function () {
+  if (!Kadira.options.enableErrorTracking) {
+    return;
+  }
+
+  let subType = null;
+  let message = null;
+  let stack = null;
+
+  if (
+    !(arguments[0] instanceof Error) &&
+    typeof arguments[0] === 'string' &&
+    typeof arguments[1] === 'string'
+  ) {
+    // Old usage
+    // Monti.trackError('type', 'error message', { stacks: 'error stack' });
+    subType = arguments[0] || 'Monti.trackError';
+    message = arguments[1];
+    stack = arguments[2] && arguments[2].stacks || '';
+  } else {
+    // New usage
+    // Monti.trackError(error, options);
+    const error = arguments[0];
+    const options = arguments[1];
+
+    message = error && error.message;
+    stack = error && error.stack || '';
```

Comment:
If there isn't a stack trace, it could create one here.

In a chromium based browser, it can use [`Error.captureStackTrace`](https://v8.dev/docs/stack-trace-api#stack-trace-collection-for-custom-exceptions) so we can prevent `Kadira.trackError` from showing in the stack trace.